apps/bot/config/config.exs

```diff
@@ -0,0 +1,9 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+use Mix.Config
+
+config :bot,
+  github_access_key: System.get_env("GITHUB_ACCESS_KEY")
```

Comment:
@ybur-yug what do you mean?  I'm not sure I follow.  You wouldn't call `Application.get_env` inside of your configuration file, what would that do?

I build my releases by injecting my ENVs into a docker container so it compiles with those values populated, saves me the hassle of maintaining the values in my systemd configure.